lib/deaf_grandma.rb

```diff
@@ -20,8 +20,29 @@ def run!
 
   def speak(input)
 
-    #Implement your code here <<<<<<<<<
+    # close the application
+    if input.upcase == "EXIT"
+      exit
+    end
+
+    # if input is UPPERCASE say one thing, if not, say another
+    if input == input.upcase
```

Comment:
see my notes about fizzbuzz. you can clean this up quite a bit without using explicit returns.